src/main/java/mods/railcraft/common/blocks/tracks/TrackSwitchBase.java

```diff
@@ -128,6 +143,56 @@ public void onNeighborBlockChange(Block block) {
         }
         super.onNeighborBlockChange(block);
     }
+    
+    /**
+     * A utility method for writing out UUID's to NBT 
+     */
+    private void writeUUIDToNBT(String key, UUID uuid, NBTTagCompound data) {
```

Comment:
I probably should have pointed this out before, but....
https://github.com/CovertJaguar/Railcraft/blob/master/src/main/java/mods/railcraft/common/util/misc/MiscTools.java#L69